bootstrap: Remove "bootstrap.btn" classes from support view buttons.

laurynmm opened the pull request:
Moves CSS styles for support view buttons to `web/styles/portico.activity.css`.

As the "sea-green" styles from the web-app were not being applied to the approve sponsorship buttons, moved the applicable CSS rules to that button class, "approve-sponsorship-button".


**Screenshots and screen captures:**

<details>
<summary>Iago user support view</summary>

| Before | After |
| --- | --- |
| ![Screenshot from 2024-01-29 12-02-45](https://github.com/zulip/zulip/assets/63245456/4cd25cd2-86f3-4eeb-b927-b1f673f4bff5) | ![Screenshot from 2024-01-29 11-59-59](https://github.com/zulip/zulip/assets/63245456/d0df87fb-98a9-4090-991d-797a4314b6b6)
</details>
<details>
<summary>remote server support view</summary>

| Before | After |
| --- | --- |
|![Screenshot from 2024-01-29 12-02-13](https://github.com/zulip/zulip/assets/63245456/5f1369eb-5da2-4830-9217-3d4463b91e80) | ![Screenshot from 2024-01-29 12-01-16](https://github.com/zulip/zulip/assets/63245456/2ef9d152-fe1b-4b94-9214-231e9b30e531)
</details>
<details>
<summary>Zulip dev support view - with sponsorship approve button</summary>

| Before | After |
| --- | --- |
| ![Screenshot from 2024-01-29 12-02-27](https://github.com/zulip/zulip/assets/63245456/5ac647b4-aa71-4a95-aaf3-9b6bf403e7f8) | ![Screenshot from 2024-01-29 12-00-40](https://github.com/zulip/zulip/assets/63245456/ace88d83-17cf-418d-83d7-6500065747f3)

</details>

---

<details>
<summary>Self-review checklist</summary>

<!-- Prior to submitting a PR, follow our step-by-step guide to review your own code:
https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code -->

<!-- Once you create the PR, check off all the steps below that you have completed.
If any of these steps are not relevant or you have not completed, leave them unchecked.-->

- [ ] [Self-reviewed](https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code) the changes for clarity and maintainability
      (variable names, code reuse, readability, etc.).

Communicate decisions, questions, and potential concerns.

- [ ] Explains differences from previous plans (e.g., issue description).
- [ ] Highlights technical choices and bugs encountered.
- [ ] Calls out remaining decisions and concerns.
- [ ] Automated tests verify logic where appropriate.

Individual commits are ready for review (see [commit discipline](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html)).

- [ ] Each commit is a coherent idea.
- [ ] Commit message(s) explain reasoning and motivation for changes.

Completed manual review and testing of the following:

- [ ] Visual appearance of the changes.
- [ ] Responsiveness and internationalization.
- [ ] Strings and tooltips.
- [ ] End-to-end functionality of buttons, interactions and flows.
- [ ] Corner cases, error conditions, and easily imagined bugs.
</details>

timabbott commented:
Posted one comment, otherwise lgtm.

timabbott commented:
Looks good to me, marked to merge once CI passes. Thanks for cleaning this up @laurynmm!

laurynmm commented:
**Updates**:
- Addressed review feedback above.
- Removed the "sea green" formatting that was added in the initial changes, as these buttons/forms will be refactored to be in a modal soon and that will likely remove the newly added colors/formatting.

<details>
<summary>Updated Iago user support view comparison between main and changes</summary>

| Before | After |
| --- | --- |
| ![Screenshot from 2024-01-29 19-30-10](https://github.com/zulip/zulip/assets/63245456/e5f68a75-a531-491c-93d3-9b3eb96cfefc) | ![Screenshot from 2024-01-29 19-29-46](https://github.com/zulip/zulip/assets/63245456/179307a5-65b9-459d-ae2b-faf08d031255)
</details>
